xpcPlugin/Timer.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2013-2018 United States Government as represented by the Administrator of the
+// National Aeronautics and Space Administration. All Rights Reserved.
+
+#ifndef Timer_hpp
```

Comment:
Should be `XPCPLUGIN_TIMER_H_` to be consistent with the rest of the plugin code.